src/mudlet.pro

```diff
@@ -219,6 +221,10 @@ macx:LIBS += \
 # will be used.
 DEFINES += LUA_DEFAULT_PATH=\\\"$${LUA_DEFAULT_DIR}\\\"
 
+# Enable the built-in updater by default. Linux packagers will find it useful to disable it
+# since package managers are responsible for updates there
+DEFINES += INCLUDE_UPDATER=TRUE
```

Comment:
I do not think you should assign a value to `INCLUDE_UPDATER` it may lead to coding errors where a coder looks for the value to be `TRUE` or `FALSE` when they should instead test for `defined(INCLUDE_UPDATER)` {which has the value of `1` if `INCLUDE_UPDATER` is defined and `0` otherwise}.

Besides which I am not exactly sure what `INCLUDE_UPDATER` will be replaced by in the preprocessor - all the escaping I had to do on the previously line to get the `LUA_DEFAULT_DIR` ASCII string value from the environment into the `LUA_DEFAULT_PATH` defined value was necessary and I am not entirely sure what `TRUE` will actually come out as...

Reply:
So you're saying it should be `DEFINES += INCLUDE_UPDATER=` ? That will hardly be intuitive to look at. Seems to be it's a pick your poison sort of thing.

Reply:
No, just `DEFINES += INCLUDE_UPDATER` then we just check for it being `#if defined(`...')' !